app/scenes/login/login.js

```diff
@@ -39,11 +58,58 @@ class Login extends Component {
         }
     }
 
-    signIn() {
+    preSignIn = () => {
+        this.setState({error: null});
```

Comment:
In case the user enters an input and then press the button it will clear any errors showing on the scene, if that makes sense